pkg/danm/danm_test.go

```diff
@@ -0,0 +1,38 @@
+package main
```

Comment:
yeah, you can't test this in other way, but just as an FYI I will detail how this should be done properly, in pratice, for future reference

So, in Golang, you can't really test the main function. also testing the main package, containing the main function, is not really preferred / good / solved / allowed. this is also why the overall coverage of the whole project hasn't increased, because you are not actually testing a package
unfortunately for you, the code is currently structured in a wrong way. more standard way would be creating a cmd directory , and relocating the "main" functions of all binaries there. and refactor the main functions, that they only contain a couple of LOCs before calling something from ...pkg/..., which is no testable

I'm gonna do this later, so no worries